app/models/student_loans.rb

```diff
@@ -1,5 +1,10 @@
-# frozen_string_literal: true
```

Comment:
I've noticed we're quite inconsistent with using frozen string literals (we didn't have it in the `MathsAndPhysics` module for example), I'm not sure if the benefits are enough to worry about but I thought I'd point it out incase you didn't mean to remove it.

Reply:
Yeah, we're definitely not consistent. I think I removed this one here so that the shape of this file more closely the source for the `MathsAndPhysics` module. Maybe I should have gone the other way and added it to `MathsAndPhysics` instead.

Reply:
I've re-instated the magic quote and added it to the other policy namespace for consistency.